### ISSUE TITLE
File Explorer fails when alias does not match hostname 

### ISSUE THREAD

**Issue by marwan-at-work** [tailscale-dev/vscode-tailscale#109]
Given the following SSH config: 

```
Host marwan
    HostName marwan
    User marwan
    ForwardAgent yes
```

The File Explorer works as expected.

However, if the alias does not match the hostname such as: 

```
Host marwan_vm
    HostName marwan
    User marwan
    ForwardAgent yes
```
Then I get the following error when clicking on "File Explorer": 

<img width="571" alt="Screenshot 2023-07-19 at 10 14 47 AM" src="https://github.com/tailscale-dev/vscode-tailscale/assets/16294261/08d70ff5-84d1-4eb3-9470-dd72245fa6f4">

**Comment by tylersmalley** [comment]
I don't believe this is expected to work, as this config would be used up when you connect to the nickname (`marwan_vm`) but not vice versa.

**Comment by tylersmalley** [comment]
Yes, we should prompt for the user - I will create an issue to track this. I don't think there is anything more we should do around using a matching HostName as we're using SSH directly and it's not a function of it.
